azure/packages/test/scenario-runner/.eslintrc.cjs

```diff
@@ -3,7 +3,10 @@
  * Licensed under the MIT License.
  */
 module.exports = {
-	extends: [require.resolve("@fluidframework/eslint-config-fluid/minimal"), "prettier"],
+	extends: [
+		require.resolve("@fluidframework/eslint-config-fluid/minimal-deprecated"),
```

Comment:
Note for reviewers: this is the most common change made to the configs - just accounting for the file name change in the base config package that was updated.